src/sentry/models/integrations/organization_integration.py

```diff
@@ -56,3 +56,32 @@ def delete(self, *args, **kwds):
             for outbox in self.outboxes_for_update():
                 outbox.save()
             super().delete(*args, **kwds)
+
+    @staticmethod
+    def services_in(config: Mapping[str, Any]) -> List[Any]:
+        return config.get("pagerduty_services", [])
+
+    def add_pagerduty_service(self, integration_key: str, service_name: str) -> Any:
+        with transaction.atomic(router.db_for_write(OrganizationIntegration)):
+            OrganizationIntegration.objects.filter(id=self.id).select_for_update()
+
+            with transaction.get_connection(
+                router.db_for_write(OrganizationIntegration)
+            ).cursor() as cursor:
+                cursor.execute(
+                    "SELECT nextval(%s)", [f"{OrganizationIntegration._meta.db_table}_id_seq"]
+                )
```

Comment:
So this is just retrieving the next sequential ID from the `organization_integration` table's primary ID right? A comment on here detailing this would be nice 😁